### ISSUE TITLE
Will there be support for nocookie in any future release?

### ISSUE THREAD

**Issue by danielatwhitegrid** [kaorun343/vue-youtube-embed#48]
Since GPDR is active in the EU, we have to take additional privacy steps, therefore my question: Will you support an option in any future release, to change the youtube embed url?

Thanks in advance,
Daniel :)

**Comment by danielatwhitegrid** [comment]
Yes, it seems like it. So I guess there are is no update on the iFrame API yet? :(

**Comment by da2x** [comment]
new YT.Player(host: 'https://www.youtube-nocookie.com');

**Comment by gerbenqikker** [comment]
I'd like to second this request

**Comment by gerbenqikker** [comment]
@danielatwhitegrid @kaorun343 @da2x
